File: Pascal/maior18.pas
program maior18;
Var
	idade :Integer;
Begin
	WriteLn('Qual sua idade?');
  Readln(idade);
	if (idade >= 18)
	Then WriteLn('Você é maior de idade!')
	Else WriteLn('Você é menor de idade');
	Readln;
End.

File: C/maior18.c
#include <stdio.h>

int main(){
    //Perguntar a idade do usuário.
    //Testa se o usuário é ou não maior de 18 anos.
    //Mostrar o resultado no terminal.
    int idade;
    
    printf("Quantos anos você tem? ");
    scanf("%d", &idade);
    if(idade >= 18){
        printf("Você é maior de idade.\n");
    }
    else {
        printf("Você ainda é menor de idade.\n");
    }
    
}